src/main/java/com/google/codeu/data/Message.java

```diff
@@ -41,6 +42,16 @@ public Message(UUID id, String user, String text, long timestamp) {
     this.timestamp = timestamp;
   }
 
+  public Message(UUID id, String user, String text, long timestamp, double score) {
```

Comment:
```suggestion
  public Message(UUID id, String user, String text, long timestamp, double sentimentScore) {
```

Change this to `sentimentScore` and on line 47 as well to be more consistent?